Added support for CDH6

jackusz opened the pull request:
The previous ImpalaJDBC41 driver would not work on CDH6 anymore. I used the newest ImpalaJDBC42 which was available. I assume it has full backward compatibility.

kupferk commented on pom.xml:
```diff
@@ -161,6 +161,49 @@
         </profile>
 
+        <profile>
+            <id>cdh6.3.3</id>
```
Can you rename the profile to CDH-6.3? This would be consistent with CDH-5.15

jackusz replied:
ok, done

jackusz commented:
> Two additional issues:
> 
> * Apparently all Travis builds fail because the new JDBC driver cannot be found: https://travis-ci.org/github/dimajix/flowman/builds/707681459 I will check if it works on my machine. Hopefully Cloudera didn't make the new driver only available to paying customers...

The driver JAR can be found in the Cloudera repo which is also added in the root pom.xml of Flowman. Shouldn't it be fetched without an issue then?
https://repository.cloudera.com/artifactory/cloudera-repos/Impala/ImpalaJDBC42/2.6.17.1020/

> * Can you please also add the new build profile to .travis.yml?

Sure, added that in my latest commit

kupferk commented:
One last change is required, otherwise the Docker image will not build:

You need to add a profile in docker/pom.xml
```
        <profile>
            <id>CDH-6.3</id>
            <properties>
                <docker.base-image.version>2.4.5</docker.base-image.version>
            </properties>
        </profile>

```

jackusz commented:
> One last change is required, otherwise the Docker image will not build:
> 
> You need to add a profile in docker/pom.xml
> 
> ```
>         <profile>
>             <id>CDH-6.3</id>
>             <properties>
>                 <docker.base-image.version>2.4.5</docker.base-image.version>
>             </properties>
>         </profile>
> ```

Ok, thanks for the tip. I've added the extra profile, let's hope that the Travis build will succeed now.
